MetaMask/metamask-extension
# Fix MMI build

In the MMI build, the build fencing in this component resulted in an `else` block with no preceding `if` block, which is a syntax error. The fence in this case was meant to include "all non-Flask build types", MMI was mistakenly missing from that list.

## Manual Testing Steps

See that `yarn lavamoat:webapp:auto --build-types mmi` passes after this change, but failed beforehand.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
